mod/gcforums/actions/gcforums/move_topic.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+$topics = str_replace(' ', '', get_input('txtTopic'));
+$forum = (int)get_input('txtForum');
+
+$error = "";
+
+// validate that the inputs are correct
+if (!preg_match("/[\d*,\s?]+/", $topics)) {
+	$error .= "<p>invalid input for topics</p>";
```

Comment:
If this is admin only it might not matter as much, otherwise maybe a translation string is needed. Few others below

Reply:
this is only for site administrator (so you and users of the site, will not have access) it has to do with the job marketplace

Reply:
Sounds reasonable